src/packages/user/current-user/modals/current-user/current-user-modal.element.ts

```diff
@@ -84,6 +84,9 @@ export class UmbCurrentUserModalElement extends UmbLitElement {
 	static override styles: CSSResultGroup = [
 		UmbTextStyles,
 		css`
+			:host {
```

Comment:
It would be better if you use the `class="uui-text"` on the first element in the render. so we avoid local color implementations. :-)

Reply:
I have tried with the class but it doesn't work. I will accept the current solution.